test/integration/configurations/nighthawk_http_origin.yaml

```diff
@@ -29,7 +29,19 @@ static_resources:
               - "*"
           http_filters:
           - name: time-tracking
+            typed_config:
+              "@type": type.googleapis.com/nighthawk.server.TimeTrackingConfiguration
+              response_options:
```

Comment:
Just noting that these yamls have become better example code through the added fields, but technically you might be able to have just `"@type": type.googleapis.com/nighthawk.server.TimeTrackingConfiguration` with no no `response_options:` inside, if there is value in keeping the test behavior identical.

Reply:
Missed this on first pass, but updated the two readme files I found that referenced dynamic-delay or time-tracking